examples/glue/glue_entrypoint.ipynb

```diff
@@ -0,0 +1,370 @@
+    "                mlm: true\n",
+    "                mlm_probability: 0.15\n",
+    "            metric_names:\n",
+    "              - LanguageCrossEntropy\n",
+    "              - MaskedAccuracy\n",
+    "\n",
+    "      # Run evaluation after every 1000 training steps\n",
+    "      eval_interval: 1000ba\n",
+    "\n",
+    "      # Use the decoupled AdamW optimizer with learning rate warmup\n",
+    "      optimizers:\n",
+    "        decoupled_adamw:\n",
+    "          lr: 5.0e-4                     # Peak learning rate\n",
+    "          betas:\n",
+    "            - 0.9\n",
+    "            - 0.98\n",
+    "          eps: 1.0e-06\n",
+    "          weight_decay: 1.0e-5           # Amount of weight decay regularization\n",
+    "      schedulers:\n",
+    "        linear_decay_with_warmup:\n",
+    "          t_warmup: 0.06dur              # Point when peak learning rate is reached\n",
+    "          alpha_f: 0.02\n",
+    "\n",
+    "      max_duration: 275184000sp          # Subsample the training data for 275M samples\n",
+    "      train_batch_size: 4000             # Number of training examples to use per update\n",
+    "      eval_batch_size: 2000\n",
+    "\n",
+    "      precision: amp                     # Use mixed-precision training\n",
+    "      grad_clip_norm: -1.0               # Turn off gradient clipping\n",
+    "      grad_accum: 'auto'                 # Use automatic gradient accumulation to avoid OOMs\n",
+    "\n",
+    "      save_folder: checkpoints           # The directory to save checkpoints to\n",
+    "      save_interval: 3500ba              # Save checkpoints every 3500 batches\n",
+    "      save_artifact_name: '{run_name}/checkpoints/ep{epoch}-ba{batch}-rank{rank}' \n",
+    "      save_num_checkpoints_to_keep: 0\n",
+    "      save_overwrite: True\n",
+    "\n",
+    "      loggers:\n",
+    "        object_store:\n",
+    "          object_store_hparams:         # The bucket to save checkpoints to\n",
+    "            s3:\n",
+    "              bucket: your-bucket-here"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fine-tuning\n",
+    "\n",
+    "If you are only fine-tuning checkpoints on the GLUE tasks, you are expected to specify the checkpoints to load from by specifying a `finetune_ckpts` list as so in the `finetune_hparams` section of your YAML. Upon runnning the entrypoint with this list, it will automatically pull all checkpoints and finetune on all of them. Note that if the `finetune_ckpts` list contains paths in object store, the entrypoint expects an `ObjectStoreLogger` and `load_object_store` instances, as well as their corresponding credentials to be specified, otherwise it will try to load from local disk. See our [checkpointing guide](https://docs.mosaicml.com/en/stable/trainer/checkpointing.html#resume-training) if you're not familiar with our checkpoint saving and loading schema. \n",
```

Comment:
```suggestion
    "If you are only fine-tuning checkpoints on the GLUE tasks, you are expected to specify the checkpoints to load from by specifying a `finetune_ckpts` list as so in the `finetune_hparams` section of your YAML. Upon running the entry point with this list, it will automatically pull all checkpoints and fine-tune on all of them. Note that if the `finetune_ckpts` list contains paths in object store, the entry point expects an `ObjectStoreLogger` and `load_object_store` instances, as well as their corresponding credentials to be specified, otherwise it will try to load from local disk. See our [checkpointing guide](https://docs.mosaicml.com/en/stable/trainer/checkpointing.html#resume-training) if you're not familiar with our checkpoint saving and loading schema. \n",
```